packages/shared/src/components/squads/PublicSquadSubmissionActions.tsx

```diff
@@ -82,7 +82,9 @@ const PublicSquadSubmissionActions = (
           onClick={onSubmit}
           type="button"
         >
-          Submit for review
+          {status === SquadStatus.Approved
```

Comment:
Is this used in the checklist? I think the idea is to remove this step from checklist, but now that I see it, not sure if we have considered what will happen with all squads that have requested but we haven't approved yet? before we release this feature?

Reply:
Currently:
If the squad _is public_: we show "Learn about public squads"
If the squad _is private_: we show "Get more traffic by going public"

Is that right?

Reply:
We should not need this item anymore: https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/1083408386/RFC-058+-+Public+Squads+GA#Public-Squad-eligibility

Reply:
We should not need any approvals anymore. Any rejected Squads in the past, shouldn't matter if they try to turn public since we won't surface public Squads anymore unless you are a member or the Squad is featured.

Reply:
@sshanzel Updated, let me know if now is ok. "Learn about squads" is always visible for admins, but once you click on "Learn more" will be marked as completed for all squads.

Reply:
Yes, once clicked, it should be completed for all Squads.

Reply:
@ilasw are we removing this thing now based on Lee's comment above? https://github.com/dailydotdev/apps/pull/3489/files#r1738285343